<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #323232;
padding-left: 4px;
spacing: 2px;
border: 1px dashed #76797C;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="l_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: #323232;
	border: none;

    margin-left: 2px;
    margin-right: 2px;
    color: #D1DBCB;
	align: RIGHT

</string>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameinp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #1e1e1e;
    selection-background-color: #1e1e1e;
    selection-color: black;
    padding: 5px;
    border-style: hidden;
	 border: 1px groove #76797C;
   
    border-radius: 2px;
    color: #eff0f1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordinp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #1e1e1e;
    selection-background-color: #1e1e1e;
    selection-color: black;
    padding: 5px;
    border-style: hidden;
	 border: 1px groove #76797C;
   
    border-radius: 2px;
    color: #eff0f1;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: #1e1e1e;
    selection-background-color: #1e1e1e;
    selection-color: black;
    padding: 5px;
    border-style: hidden;
	 border: 1px groove #76797C;
   
    border-radius: 2px;
    color: #eff0f1;
}

QPushButton:disabled
{
    background-color: #323232;
    border-width: 1px;
    border-color: #454545;
    border-style: solid;
    padding-top: 5px;
    padding-bottom: 5px;
    padding-left: 10px;
    padding-right: 10px;
    border-radius: 2px;
    color: #454545;
}

QPushButton:focus {
    background-color: #D1DBCB;
    color: black;
}

QPushButton:pressed
{
    color: black;
    background-color: #D1DBCB;
    padding-top: -15px;
    padding-bottom: -17px;
}</string>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_username">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: #323232;
	border: none;

    margin-left: 2px;
    margin-right: 2px;
    color: #D1DBCB;


</string>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateinfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: #1e1e1e;
    selection-background-color: #1e1e1e;
    selection-color: black;
    padding: 5px;
    border-style: hidden;
	 border: 1px groove #76797C;
   
    border-radius: 2px;
    color: #eff0f1;
}

QPushButton:disabled
{
    background-color: #323232;
    border-width: 1px;
    border-color: #454545;
    border-style: solid;
    padding-top: 5px;
    padding-bottom: 5px;
    padding-left: 10px;
    padding-right: 10px;
    border-radius: 2px;
    color: #454545;
}

QPushButton:focus {
    background-color: #D1DBCB;
    color: black;
}

QPushButton:pressed
{
    color: black;
    background-color: #D1DBCB;
    padding-top: -15px;
    padding-bottom: -17px;
}</string>
    </property>
    <property name="text">
     <string>UPDATE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
